src/osd/osd_types.h

```diff
@@ -3986,7 +3986,7 @@ struct object_info_t {
 
   flag_t flags;
 
-  static string get_flag_string(flag_t flags) {
+  string get_flag_string(flag_t flags) const {
```

Comment:
`static` is more restrictive than `const`. `get_flag_string()` does not access any non-static member variable. so we'd better prefer `static` over `const`.

> since the functions with similar usage don't use static flag.

i am not convinced by this.